css/cart.css

```diff
@@ -0,0 +1,88 @@
+.cart-content {
+  margin: 0 5px;
+  padding-top: 3.5rem;
+}
+.cart-section {
+  display: flex;
+  justify-content: space-evenly;
+}
+.cart-products {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  margin: 1rem 1rem 1rem 10rem;
+}
+
+.cart-products > * {
+  margin: 2rem;
+}
+
+.cart-card {
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  flex-wrap: wrap;
+  background-color: var(--secondary-color);
+  color: var(--white);
+  padding: 1rem;
+  width: 50%;
+  min-width: 240px;
+  max-width: 500px;
+}
+
+.card-details {
+  margin-top: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.quantity {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
```

Comment:
as per UI , you can give padding-right so the buttons and text "Quantity"have space around to look batter.